app/assets/v2/js/open-chat.js

```diff
@@ -0,0 +1,14 @@
+const openChat = () => {
+  $('[data-openchat]').each(function(index, elem) {
+    let chatWindow;
+
+    $(elem).on('click', function() {
+      let user = $(elem).data('openchat');
+      let url = user ? `https://chat.gitcoin.co/gitcoin/messages/@${user}` : 'https://chat.gitcoin.co/';
+
+      chatWindow = window.open(url, 'Loading', 'top=0,left=0,width=400,height=600,status=no,toolbar=no,location=no,menubar=no,titlebar=no');
```

Comment:
we could do just a `window.open` ! what do you think ?

Reply:
what do you mean?

Reply:
oo without the variable declaration? Is because I want to reuse the window to avoid multiple popups

Reply:
gotcha